service/src/main/java/greencity/service/EcoNewsCommentServiceImpl.java

```diff
@@ -173,7 +174,8 @@ public void deleteById(Long id, UserVO userVO) {
         comment.setStatus(CommentStatus.DELETED);
         String accessToken = httpServletRequest.getHeader(AUTHORIZATION);
         CompletableFuture.runAsync(
-            () -> ratingCalculation.ratingCalculation(RatingCalculationEnum.DELETE_COMMENT, userVO, accessToken));
+            () -> ratingCalculation.ratingCalculation(RatingCalculationEnum.DELETE_COMMENT_OR_REPLY, userVO,
+                accessToken));
```

Comment:
is it possible to get userVo and accessToken in the service instead of passing it each time?